timeless/restaurants/tables/forms.py

```diff
@@ -2,7 +2,7 @@
 from timeless.restaurants import models
 
 
-class TableForm(forms.BaseModelForm):
+class TableForm(forms.ModelForm):
```

Comment:
@vryazanov Tell me pls what is difference?

Reply:
@vladarefiev `ModelForm` can `create` / `update` instances